tests/commands/envelope_test.py

```diff
@@ -355,3 +356,41 @@ def test_apply_from_user_and_email(self):
 
         self.assertTrue(env.sign)
         self.assertIs(env.sign_key, mock.sentinel.gpg_key)
+
+
+class TestSendCommand(unittest.TestCase):
```

Comment:
Oh I didn't think about that!
It seems that the tests you refer to actually just wrap the test-methods in a `@inlineCallbacks`, which makes them return deferreds themselves. I will add those here as well, this way we are at least consistent.

As a side-note: I wouldn't mind at all dropping twisted for asyncio, which is even part of the standard lib in python3 I think. But making such a big change should be topic for another time..